### PR TITLE
Add audit-logger component

### PR DESCRIPTION
**Pull request by prayerslayer**
This way we can get rid of the default audit logger.

### CODE REVIEW

**Comment by harti2006** [comment]
:+1:

File: src/org/zalando/stups/kio/audit.clj [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  []
+  (tf/unparse date-formatter (t/now)))
+
+(defn drop-nil-values
+  [record]
+  (into {} (remove (comp nil? second) record)))
+
+(defn app-modified
+  [tokeninfo app]
+  {:event_type   {:namespace "internal"
```
Comment (harti2006) [inline src/org/zalando/stups/kio/audit.clj]: cloud.zalando.com

**Comment by harti2006** [comment]
:+1:

**Comment by prayerslayer** [comment]
:-1:

**Comment by prayerslayer** [comment]
Uses `:date-time-no-ms` formatter!

**Comment by ChristianLohmann** [comment]
👍

**Comment by prayerslayer** [comment]
👍

**Pull request merged**